src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
+
+    private:
```

Comment:
It's not specifically addressed in the coding guidelines, but all other header files seem to have separate `private:` blocks for function and member variable declarations.